conf/machine/include/imx-base.inc

```diff
@@ -445,6 +445,8 @@ PREFERRED_VERSION_ffmpeg:mx8-nxp-bsp                    ??= "4.4.1"
 SOC_HAS_VIVANTE_KERNEL_DRIVER_SUPPORT        = "0"
 SOC_HAS_VIVANTE_KERNEL_DRIVER_SUPPORT:imxgpu = "1"
 
+DISTRO_FEATURES:remove:mx8mm-nxp-bsp = "vulkan"
```

Comment:
We need to handle this in the mesa bbappend file, not removing this in distro.

Reply:
So I think we cannot do the removal. We cannot vary distro features as a giant number of recipes would need to become socarch.

Reply:
Then I will drop this patch in favor of adding it to a bunch of bbappends.

Reply:
Maybe we should revert the vulkan removal and NXP can rework for it avoid creating the surface for Mini. @thochstein ?

Reply:
I will look into what can be done. What I wonder is whether the library already fails at runtime, and maybe the only problem was my attempt to exclude it.

Reply:
> Maybe we should revert the vulkan removal and NXP can rework for it avoid creating the surface for Mini. @thochstein ?

Is this expected to work? If a package is configured to use Vulkan can we really expect the package to work when the libary fails because Vulkan is not supported?

Reply:
Besides, the whole point of excluding Vulkan support was so the user would get a build time failure instead of runtime. Isn't that more important?

Should there be a `vulkan` machine feature?

Reply:
Not necessarily, assuming the driver don't allow the misuse of vulkan. I'll ping you for a short call about it tomorrow ;-)

Reply:
@otavio @thochstein Let me know how I can help. I now dropped the `remove vulkan from distro features` patch from the PR as it anyway is not the solution.